lib/pleaserun/platform/base.rb

```diff
@@ -130,6 +130,13 @@ class InvalidTemplate < ::StandardError; end
     validate { |v| v == "ulimited" || v.to_i > 0 }
   end
 
+  attribute :envs, "Env key/value pairs to insert into init script (Upstart only)" do
```

Comment:
Note: we could do something similar in the `sysv` and `systemd` platforms by using escaped `export` calls (within the chroot subshell) and the `Environment` directive respectively.